app/views/components/_contents-list.html.erb

```diff
@@ -11,7 +12,13 @@
                               end
 %>
 <% if contents.any? %>
-  <nav role="navigation" class="app-c-contents-list">
+  <nav
+    role="navigation"
+    class="app-c-contents-list"
+    <% if aria_label.present? %>
```

Comment:
This could be written as

aria_label ||= false

<%= "aria-label=#{aria_label}" if aria_label %>

Reply:
I gave it a try but your version builds 
<... aria-label="firstword" every other word ...>
I tried quoting it to avoid but I couldn't get it working and it was starting to look messy.